mints/mapping.txt

```diff
@@ -1,8 +1,14 @@
 token_id-"component_id"
-X-
+X-"contract/valory/mech/0.1.0"
+X-"contract/valory/erc20/0.1.0"
+X-"contract/valory/market_maker/0.1.0"
```

Comment:
This name could collide with some components in the market-creator repository (e.g., agent/valory/market_maker/0.1.0, service/valory/market_maker:0.1.0)

Reply:
Are they the same or different contracts?